### DISCUSSION TITLE
Is kwok used in any real world cases? how many node and pods managed by kwok?

### DISCUSSION THREAD

**Discussion by zwpaper** [kubernetes-sigs/kwok#291] [General]
as described in the title, I was curious about the performance and real-world use cases of kwok.

when I run kwok with hundreds of nodes, plenty of nodes become NotReady.

are there any performance data or showcases we have?

**Comment by wzshiming** [comment]
These are some of the cases I know of

- https://karmada.io/docs/developers/performance-test-setup-for-karmada/
- https://medium.com/@sunyanan.choochotkaew1/going-beyond-limits-scalability-test-ci-for-kubernetes-cni-operator-with-simulated-cluster-ed53e772dfa5
- https://cloud.tencent.com/developer/article/2210383


> when I run kwok with hundreds of nodes, plenty of nodes become NotReady.

This will be tracked in #251 to optimize

> are there any performance data or showcases we have?

The performance report will be completed before the v1 release, but there are no plans for v1 at this time.
